lib/widgets/components/tickets/tickets_section.dart

```diff
@@ -14,64 +13,59 @@ class TicketSection extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BlocProvider.value(
-      value: sl.get<TicketsCubit>(),
-      child: Column(
-        children: [
-          BlocConsumer<TicketsCubit, TicketsState>(
-            listener: (context, state) {
-              if (state is TicketUsing) {
-                // Remove the swipe overlay
-                Navigator.of(context, rootNavigator: true).pop();
-                // TODO consider using a nicer loading indicator
-                LoadingOverlay.of(context).show();
-              } else if (state is TicketUsed) {
-                final envState = context.read<EnvironmentCubit>().state;
-                LoadingOverlay.of(context).hide();
-                ReceiptOverlay.of(context).show(
-                  receipt: state.receipt,
-                  isTestEnvironment: envState is EnvironmentLoaded &&
-                      envState.isTestEnvironment,
-                );
+    return Column(
+      children: [
+        BlocConsumer<TicketsCubit, TicketsState>(
+          listener: (context, state) {
+            if (state is TicketUsing) {
+              // Remove the swipe overlay
+              Navigator.of(context, rootNavigator: true).pop();
+              // TODO consider using a nicer loading indicator
+              LoadingOverlay.of(context).show();
+            } else if (state is TicketUsed) {
+              final envState = context.read<EnvironmentCubit>().state;
+              LoadingOverlay.of(context).hide();
+              ReceiptOverlay.of(context).show(
+                receipt: state.receipt,
+                isTestEnvironment:
+                    envState is EnvironmentLoaded && envState.isTestEnvironment,
+              );
+            }
+          },
+          builder: (context, state) {
+            if (state is TicketsLoading) {
+              return const Text('loading');
```

Comment:
We should refer this string from the String Constants class

Reply:
Replaced with shimmering effect